tests/test_did_meta_plugins.py

```diff
@@ -198,7 +198,9 @@ def mongo_meta():
         host='mongo',
         port=27017,
         db='test_db',
-        collection='test_collection'
+        collection='test_collection',
+        user="rucio",
+        password="mongo-meta",
```

Comment:
It would be great if we had another `mongo_meta` fixture without `user` and `password`, to ensure that unauthenticated mongodb connections still work

Reply:
That would need a second mongodb in the docker compose yaml I guess?

Reply:
I added the second mongo definition and parametrized the test to test both authenticated and unauthenticated database connections